src/xrServerEntities/script_ini_file.h

```diff
@@ -8,7 +8,12 @@
 
 #pragma once
 
+#include "pch_script.h"
 #include "script_token_list.h"
+#include "xrScriptEngine/Functor.hpp"
+
+using namespace luabind;
+using namespace luabind::policy;
```

Comment:
Never use `using namespace` in the global headers.

Reply:
Fixed